Clarify "Edit me" message on docs site

SiriusStarr opened the pull request:
There is currently a link with the text "Edit me" in the header of the docs website; clicking on it opens the file for editing on github.  If we're not going to add an extra header, it should be removed.  (Unless someone has something to go in the header?)

![Screenshot_2019-12-10 Cheatsheet — Dhall documentation](https://user-images.githubusercontent.com/2049163/70513051-b7b2c200-1ae5-11ea-987f-3d3fc560e730.png)

Gabriel439 commented:
@SiriusStarr: It's not clear to me why you're proposing to remove the "Edit me" link.  As far as I can tell the "Edit me" link works

SiriusStarr commented:
@Gabriel439 I was laboring under the mistaken assumption that it was intended to be a static docs site, not a wiki of sorts, since "Edit me" sounds like the placeholder text that would be left by someone overlooking something they were supposed to fill in during configuration.

I'm updating this PR correspondingly.

sjakobi commented on docs/_templates/layout.html:
```diff
@@ -4,7 +4,7 @@
   <a style="text-decoration: none"
      href="https://github.com/dhall-lang/dhall-lang/edit/master/docs/{{ sourcename | replace('.txt', '') }}">
-  Edit me
+  Contribute to this page
```
IMHO "Edit me" is actually a bit clearer than "Contribute to this page".

How about this?

```suggestion
  Edit me on GitHub
```

SiriusStarr replied:
Hm, my point is that if the intent is to be a wiki, the message should be clear that it's actively *soliciting* user contributions rather than something that looks like an internal configuration message that's accidentally showing (as I took it at first glance).

sjakobi replied:
Hmm. Maybe we can improve the styling a bit? And add an exclamation mark to emphasize we invite contributions?!

SiriusStarr replied:
`Want to contribute?  Edit me on Github!`  Or something along those lines?

SiriusStarr commented:
Per conversation with @sjakobi , I've updated the message to `Want to contribute? Edit me on Github!` (and also updated the reference to the message elsewhere).  Are people good with this, or is there a preference to leave it as it was?
